src/Jackett.Common/Definitions/shareuniversity.yml

```diff
@@ -0,0 +1,143 @@
+---
+  site: ShareUniversity
+  name: ShareUniversity
+  description: "ShareUniversity is a Private Torrent Tracker for High Quality Nordic Content"
+  language: en-us
+  type: private
+  encoding: UTF-8
+  links:
+    - https://shareuniversity.org
+
+  caps:
+    categorymappings:
+      - {id: 1, cat: Movies}
+      - {id: 8, cat: Movies}
+      - {id: 2, cat: TV}
+      - {id: 9, cat: TV}
+      - {id: 12, cat: TV/Anime}
+      - {id: 6, cat: Other}
+      - {id: 13, cat: Audio/Audiobook}
+      - {id: 4, cat: Books/Ebook}
+      - {id: 16, cat: E-learning}
+      - {id: 5, cat: PC/Games}
+      - {id: 11, cat: Movies}
+      - {id: 3, cat: Audio}
+      - {id: 10, cat: XXX}
+      - {id: 7, cat: Movies}
```

Comment:
category 7 can not be both tv and movies

Reply:
Removed TV, just left it as Movies